Resources/Prototypes/Entities/Markers/atmos_blocker.yml

```
@@ -108,3 +108,16 @@
   components:
   - type: AtmosFixMarker
     mode: 7
+
+- type: entity
+  id: AtmosFixCoolant
+  suffix: Coolant
+  description: "Supermatter coolant, equivalent to a liquid nitrogen canister"
+  components:
+  - type: AtmosFixMarker
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)